### PR TITLE
Began F words page 308(285)

### PR DESCRIPTION
**Pull request by haith4m**

### CODE REVIEW

File: Somali-Dictionary/alphabetical-order/F words Qaamuus.txt [hunk 1]
```diff
@@ -462,4 +462,168 @@ waxyaalaha foosha xun ee diintu reebtay.
...
+           2. Taag.
+
+fadeexad m.dh ld fadeexo.
+
+fadeexayn m.f.dh eeg fadeexee.
+
+fadeexee f.g2 (-xeeyay, -xaysay) Cid fadeexo
+ku samayn; dacaayadeyn. 
+
 Completed page 310(287)
```
Comment (haith4m) [inline Somali-Dictionary/alphabetical-order/F words Qaamuus.txt:629]: Made a big mistake. It should be page 311(288)

Comment (yahyaqowle) [reply Somali-Dictionary/alphabetical-order/F words Qaamuus.txt]: Have u checked how to revert commits and also what you can do is, you can copy all the text(pages) u did in f words and paste another txt file,then go back in time e.g when you began F words and paste all the pages or texts you did and commit a new commit with a message like "F words finished page 100(**) - page 110(**)" this is example but try. Good luck

Comment (haith4m) [reply Somali-Dictionary/alphabetical-order/F words Qaamuus.txt]: Thanks wll. Im in college today I will do it when I go home.

**Review by yahyaqowle — COMMENTED** [review]
Im merging the pull request but I will re do it again.

**Pull request merged**